src/pages/TradeSummary/index.tsx

```diff
@@ -138,7 +138,7 @@ const TradeSummary: React.FC<TradeSummaryProps> = ({ history, location }) => {
                         <div className="transaction-info-date">
                           {transaction && <span>{transaction.blockTime?.format('DD MMM YYYY HH:mm:ss')}</span>}
                           {transaction ? (
-                            <span>{fromSatoshiFixed(transaction.fee, 8, 8)} Fee</span>
+                            <span>{fromSatoshiFixed(transaction.fee.toString(), 8, 8)} Fee</span>
```

Comment:
why we need to change this now?

Reply:
After fixing Reselect typing bug this TS typing error appeared. fromSatoshiFixed expect a string and transaction.fee is a number

Reply:
I double checked and it doesn't break things